composer.json

```diff
@@ -19,7 +19,8 @@
         "drupal/schemata": "1.x-dev#8325d172e1d6880aa24073f8f751ef089282cf9a",
         "drupal/openapi": "1.x-dev#e8a82f87dbbb83dc89f9455f7a2e5ba6c6d2cdff",
         "drupal/jsonapi": "1.0.0",
-        "drupal/simple_oauth": "2.0-rc2"
+        "drupal/simple_oauth": "2.0-rc2",
+        "webflo/drupal-core-strict": "^8.3.3"
```

Comment:
Can we move this to just below `"drupal/core": "~8.3.3"? Then it'll be easier to not forget to keep them in sync.